src/main/java/org/fundacionjala/core/selenium/interaction/GuiInteractioner.java

```diff
@@ -218,4 +222,65 @@ public static String getValueOfWebElement(final By by) {
     public static void waitLoadPage(final WebElement webElement) {
         WebDriverManager.getInstance().getWebDriverWait().until(ExpectedConditions.visibilityOf(webElement));
     }
+
+    /**
+     * [SL] Sends the key down simulating a scroll.
+     *
+     * @param scroll
+     */
+    public static void loadDynamicTable(final By scroll) {
```

Comment:
Change the name of this method for another more descriptive, please (it should not be useful only for DynamicTables, true?)